app/controllers/finance/npq/course_payment_breakdowns_controller.rb

```diff
@@ -6,20 +6,12 @@ class CoursePaymentBreakdownsController < BaseController
       def show
         @npq_lead_provider = lead_provider_scope.find(params[:lead_provider_id])
         @cpd_lead_provider = @npq_lead_provider.cpd_lead_provider
-
-        @statement = Finance::Statement::NPQ.find_by(
-          name: params[:statement_id],
-          cpd_lead_provider: @cpd_lead_provider,
-        )
-
-        if @statement.name == "January 2022"
-          @statement.id = nil
-        end
-
         @npq_course        = NPQCourse.find_by!(identifier: params[:id])
+        @statement         = @cpd_lead_provider.npq_lead_provider.statements.find(params[:statement_id])
         @breakdown         = Finance::NPQ::CalculationOrchestrator.new(
           statement: @statement,
           contract: @npq_lead_provider.npq_contracts.find_by!(course_identifier: params[:id]),
+          aggregator: ParticipantEligibleAndPayableAggregator,
```

Comment:
only using `ParticipantEligibleAndPayableAggregator` here will cause several bugs:
- when the declarations become `paid`, those declarations will drop out of scope and the output fees fees will drop to zero
- when viewing this page for a statement that has declarations assigned to it eg after payment freeze, it will ignore all those assigned declarations and therefore show incorrect values

Reply:
@asmega we'll address this in the next sprint, Conor and I have created the stories and we'll refine them tomorrow